sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
```

Comment:
This method of sorting has a time complexity of O(n^2). You might be able to improve it to have O(nlogn) time complexity by implementing quicksort